File: BOG/python/2869.py
import sys

sys.stdin = open('example.txt', 'r')

input = list(map(int,sys.stdin.readline().split()))

speed= input[0] # A
sleep = input[1] # B
target = input[2] # V

# location = 0
up = 0
down = 0
day = 0


while True:
    day += 1
    up = speed * day
    if up - down >= target:
        break
    down = sleep * day


    

# while location <= target:
#     location += speed
#     location -= sleep
#     day += 1

print(day)

File: BOG/javascript/2869.js
const fs = require("fs");
let filepath = process.platform === "linux" ? "/dev/stdin" : "example.txt";
let input = fs.readFileSync(filepath).toString().trim();

let [a, b, v] = input.split(" ").map(el => Number(el));

let curLoc = 0;
let day = 0;

while (true) {
    day += 1;   // 낮에 시작하므로 먼저 일수를 증가시킵니다.
    curLoc += a;  // 낮 동안 올라갑니다.
    if (curLoc >= v) break;  // 목표 높이에 도달하면 반복을 종료합니다.
    curLoc -= b;  // 밤에 미끄러집니다.
}

console.log(day);  // 최종적으로 필요한 일수를 출력합니다.
